androidapp/HopSquadApp/app/src/main/java/com/hopsquad/hopsquadapp/TapListFragment.java

```diff
@@ -0,0 +1,57 @@
+package com.hopsquad.hopsquadapp;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class TapListFragment extends Fragment {
```

Comment:
What's the difference between a Fragment and an Activity?

Reply:
Fragments are components of an android app that are attached to an activity and that are aware of its cycle. Frequently used to divide UI and logic specific to that UI in its own set of files, to later be composed in an activity. In our case the MainActivity holds a fragment at a time. The user selects an option in the bottom navigation bar and a fragment is switched into place. Each fragment is responsible of inflating its layout and responding to user input.